epersongroups.md

```diff
@@ -300,3 +333,9 @@ This supports a basic search in the metadata.
 It will search in:
 * UUID (exact match)
 * group name
+
+## Related DSpace Object of group
+**GET /api/eperson/groups/<:uuid>/object** (READ-ONLY)
```

Comment:
which is the use case behind this endpoint? right now is true that a "community administrator" group cannot be used elsewhere but this is not really enforced at the JAVA API layer so I don't know if it is a good idea to "lock" that here now if we don't have a concrete need

Reply:
This is for the edit groups UI which displays the community or collection:
<img width="866" alt="Schermafdruk 2020-03-11 09 06 46" src="https://user-images.githubusercontent.com/1528033/76395495-bb214800-6377-11ea-898d-face9d6101ba.png">

Reply:
ok thanks